specification/web/resource-manager/Microsoft.Web/stable/2021-03-01/Certificates.json

```diff
@@ -0,0 +1,710 @@
+              "description": "Specification for the App Service Environment to use for the certificate.",
+              "readOnly": true
+            },
+            "keyVaultId": {
+              "description": "Key Vault Csm resource Id.",
+              "type": "string"
+            },
+            "keyVaultSecretName": {
+              "description": "Key Vault secret name.",
+              "type": "string"
+            },
+            "keyVaultSecretStatus": {
+              "description": "Status of the Key Vault secret.",
+              "enum": [
+                "Initialized",
+                "WaitingOnCertificateOrder",
+                "Succeeded",
+                "CertificateOrderFailed",
+                "OperationNotPermittedOnKeyVault",
+                "AzureServiceUnauthorizedToAccessKeyVault",
+                "KeyVaultDoesNotExist",
+                "KeyVaultSecretDoesNotExist",
+                "UnknownError",
+                "ExternalPrivateKey",
+                "Unknown"
+              ],
+              "type": "string",
+              "readOnly": true,
+              "x-ms-enum": {
+                "name": "KeyVaultSecretStatus",
+                "modelAsString": false
+              }
+            },
+            "serverFarmId": {
+              "description": "Resource ID of the associated App Service plan, formatted as: \"/subscriptions/{subscriptionID}/resourceGroups/{groupName}/providers/Microsoft.Web/serverfarms/{appServicePlanName}\".",
+              "type": "string"
+            },
+            "canonicalName": {
+              "description": "CNAME of the certificate to be issued via free certificate",
+              "type": "string"
+            },
+            "domainValidationMethod": {
+              "description": "Method of domain validation for free cert",
+              "type": "string"
+            }
+          },
+          "x-ms-client-flatten": true
+        }
+      }
+    },
+    "CertificateCollection": {
+      "description": "Collection of certificates.",
+      "required": [
+        "value"
+      ],
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "Collection of resources.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Certificate"
+          }
+        },
+        "nextLink": {
+          "description": "Link to next page of resources.",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "CertificatePatchResource": {
+      "description": "ARM resource for a certificate.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "./CommonDefinitions.json#/definitions/ProxyOnlyResource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "description": "CertificatePatchResource resource specific properties",
+          "type": "object",
+          "properties": {
+            "password": {
```

Comment:
Please set x-ms-secret to true.

Reply:
This property was never annotated with x-ms-secret in all previous versions - double checking with @ElleTojaroon if there is a reason since adding this annotation will cause the value to not be returned in responses